File: benchmarks/NSubstitute.Analyzers.Benchmarks.CSharp.Source/DiagnosticsSources/UnusedReceivedDiagnosticsSource.cs
using NSubstitute.Analyzers.Benchmarks.Shared.Models;

namespace NSubstitute.Analyzers.Benchmarks.CSharp.Source.DiagnosticsSources
{
    public class UnusedReceivedDiagnosticsSource
    {
        public void NS5000_ReceivedCheck()
        {
            var substitute = Substitute.For<IFoo>();

            substitute.Received(1);
            SubstituteExtensions.Received(substitute, 1);
        }
    }
}

File: benchmarks/NSubstitute.Analyzers.Benchmarks.VisualBasic.Source/DiagnosticsSources/UnusedReceivedDiagnosticsSource.vb
Imports NSubstitute.Analyzers.Benchmarks.Shared.Models

Namespace DiagnosticsSources
    Public Class UnusedReceivedDiagnosticsSource
        Public Sub NS5000_ReceivedCheck()
            Dim substitute = NSubstitute.Substitute.[For](Of IFoo)()
            substitute.Received(1)
            SubstituteExtensions.Received(substitute, 1)
        End Sub
    End Class
End Namespace
